packages/backend-core/src/context/index.ts

```diff
@@ -67,6 +67,7 @@ export const getTenantIDFromAppID = (appId: string) => {
 
 // used for automations, API endpoints should always be in context already
 export const doInTenant = (tenantId: string | null, task: any) => {
+  tenantId = tenantId || DEFAULT_TENANT_ID
```

Comment:
I'd be very wary of shipping this, if there is every a scenario where we supply a null or undefined tenant id (which really should be addressed) we will silently go to the default db instead. In a single tenant environment this is ok, but in cloud this could cause havoc.